### PR TITLE
Added QUIRC to libs

### PR DESCRIPTION
**Pull request by cxziaho**
QR Decoding library.  Example usage [here](https://github.com/cxziaho/qrdemo)

### CODE REVIEW

**Comment by devnoname120** [comment]
Please:
- Add it to `travis_packages.sh`
- Create a pull request to add it as well on vitasdk/vdpm

**Comment by cxziaho** [comment]
Done :)  I assume for vitasdk/vdpm I add it to `install-all.sh`

**Comment by devnoname120** [comment]
Yes, this is it. 😄

**Comment by cxziaho** [comment]
I think the problem is my varying uses of library name `quirc` and repo name `quirc-vita`.  When it downloads the file, it tries to open the folder quirc when the repo is saved to quirc-vita (so it can't cd in).  Is there any way to rename the folder it clones into, or should I just rename my repo and change the URL.

File: quirc/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,17 @@
+pkgname=quirc
+pkgver=9999
+pkgrel=1
+url="https://github.com/cxziaho/quirc-vita"
+source=("git://github.com/cxziaho/quirc-vita.git")
+sha256sums=('SKIP')
+
+build() {
+  cd $pkgname
```
Comment (devnoname120) [inline quirc/VITABUILD]: The issue is here, it should be `$pkgname-vita`.

File: quirc/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  mkdir build && cd build
+  make
+}
+
+package () {
+  cd $pkgname/build
```
Comment (devnoname120) [inline quirc/VITABUILD]: Here as well.

**Comment by devnoname120** [comment]
We prefer packages to use a tarball + patch because it gives us a better control over them.

I suggest you to do a commit to revert the removal of the useless files (it's not needed and it adds complexity for maintaining), rebase with `git rebase -i HEAD~2` to merge the two quirc-vita commits into one, and then do a patch with `git format-patch master --stdout > quirc.patch`.

You can check the package [opus](https://github.com/vitasdk/packages/tree/ba01cb4df29b10bd444a834b6dac81b1209108f8/opus) to see how it's done.

**Comment by devnoname120** [comment]
Now that I think of it, it would be easier just to use upstream (preferably a tarball) and add your modified Makefile in your pull request.

**Comment by cxziaho** [comment]
Done (I hope).  Using the tarball from quirc release v1.0, and supplying my own Makefile to build with.  Hopefully this build succeeds...

**Comment by cxziaho** [comment]
Where is the tar.gz being extracted too?  I would have thought it would have been `v1.0`(the same name as the archive)

**Comment by devnoname120** [comment]
It's extracted to `quirc-1.0` since that's the name of the directory inside the `.tar.gz` file.

File: quirc/VITABUILD [hunk 3]
```diff
@@ -0,0 +1,18 @@
+pkgname=quirc
+pkgver=1.0
+pkgrel=1
+url="https://github.com/dlbeer/quirc/"
+source=("https://github.com/dlbeer/quirc/archive/v${pkgver}.tar.gz" "Makefile")
+sha256sums=('SKIP')
```
Comment (devnoname120) [inline quirc/VITABUILD]: sha256sums=('8141637f55b1cb50234689d91b85ad8ac21c0e5c' 'SKIP')

**Comment by devnoname120** [comment]
Note that you can make sure that your package compiles without waiting for Travis by executing [`vita-makepkg`](https://github.com/vitasdk/vita-makepkg) in the directory of your package.

**Comment by devnoname120** [comment]
Merged in https://github.com/vitasdk/packages/commit/43d7fe5995dfffc8123f9299c71a1c1deaa1f78f.

**Pull request closed without merging**